### PR TITLE
Second Readme commit

### PR DESCRIPTION
**Pull request by anapereda**
Please, someone, look at the changes made after Girija's feedback! 

See you in a bit!

### CODE REVIEW

**Review by GirijaJoshi — APPROVED** [review]
Ana yes you included things, couple more suggestions
1. Approximately 30% of all new diagnosis will be breast cancer. (can you change is to  30% of all new **cancer** diagnosis...), so we know that we are talking about only cancer

2. Other statistics reveal that 1 in 8 women will be diagnosed through their lifetime. In this sentence 1 in 8 women diagnosed with what? you have to mention Breast Cancer

3. Upcoming tasks are paragraph type can you make bullet, each task will be in new line.

File: README.md [hunk 1]
```diff
@@ -30,14 +30,16 @@ All features are encoded; documentation [link:] (https://www.bcsc-research.org/d
...
 [Breast Histopathology Images](https://www.kaggle.com/paultimothymooney/breast-histopathology-images/)
 
-The original dataset was formed of 162 whole mount slides of Breast Cancer specimen, 277534 oatches of 50 x 50 images were extracted; 198,738 IDC negative and 78,786 IDC positive. The patients file name is in the format:
+The original dataset was formed of 162 whole mount slides of Breast Cancer specimen, 277534 patches of 50 x 50 images were extracted; 198,738 IDC negative and 78,786 IDC positive. The patients file name is in the format:
     uxXyYclassC.png → 10253idx5
 X and Y are the coordinates where the patch were cropped from and C indicates class: 0 is non-IDC and 1 is IDC
 
 ## Questions ##
 
 * Understanding risk factors distributions to promote early detection practices; breast self exam, clinical breast exam and mammogram
```
Comment (hkyopreston) [inline README.md:41]: Phrase as a question: What are the risk factors that can promote early detection practices? Do certain risk factors warrant a breast self exam, clinical breast exam, or mammogram?

File: README.md [hunk 2]
```diff
@@ -30,14 +30,16 @@ All features are encoded; documentation [link:] (https://www.bcsc-research.org/d
...
+The original dataset was formed of 162 whole mount slides of Breast Cancer specimen, 277534 patches of 50 x 50 images were extracted; 198,738 IDC negative and 78,786 IDC positive. The patients file name is in the format:
     uxXyYclassC.png → 10253idx5
 X and Y are the coordinates where the patch were cropped from and C indicates class: 0 is non-IDC and 1 is IDC
 
 ## Questions ##
 
 * Understanding risk factors distributions to promote early detection practices; breast self exam, clinical breast exam and mammogram 
-*Deep Neural Machine Learning Model to analyze mammograms and detect Breast Cancer
+* What are some of the most important risk factors to the development of breast cancer
+* Deep Neural Machine Learning Model to analyze mammograms and detect Breast Cancer
```
Comment (hkyopreston) [inline README.md:42]: Phrase as a question: Can a Deep Neural Machine Learning Model analyze mammogram images in order to detect Breast Cancer?

**Review by hkyopreston — DISMISSED** [review]
The only comments I have were to rephrase the comments under the Questions section as actual Questions.

**Comment by anapereda** [comment]
not needed

**Comment by anapereda** [comment]
final changes review from the team- Ready for submission (segment 1)

**Review by GirijaJoshi — APPROVED** [review]
good

**Review by tienl — APPROVED** [review]

**Review by tienl — APPROVED** [review]

**Pull request merged**